### DISCUSSION TITLE
Show Ingredients only once

### DISCUSSION THREAD

**Discussion by mptr8956** [TandoorRecipes/recipes#2237] [Q&A]
Hi,
is it possible to have the ingredients list only once (on top) and not on top & with every step? That feels somewhat unnatural. Can that be turned off? I found out that if you put everything in the first step, it will show ingrendients only once but as soon as i add a second step, it shows the ingredients in the summary on top & the first step.

Is that by design or can that be changed?

**Comment by vabene1111** [comment]
can be disabled in the additional options of a recipe

**Comment by mptr8956** [comment]
Hi, thanks for your answer. But I want it the other way around (just show the one on the top and nothing else in the recipe steps). Is that possible as well?

**Reply by vabene1111** [reply]
currently not, this could probably be added in the future

**Comment by ModestTG** [comment]
+1 for implementing this feature. I would like to see the ingredients at the top, and not in each recipe step. Perhaps a way to just define the recipe ingredients.

**Reply by vabene1111** [reply]
just define will not be added since it will require a completely new data model but you can just add them to the first step. 

also btw. if you only use one step ingredients are only shown on top already.

**Comment by MiggNerd** [comment]
I can't find the place to disable this option in a recipe.
Please can anyone tell me where it is?

**Comment by SunshineTraveller** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我已收到您的邮件...

**Comment by Nattfisk** [comment]
Being able to hide the ingredients from steps, and only keep the overview at the top would be a great feature indeed!
